test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -28,7 +28,7 @@
     private static final List<ReadOnlyPerson> EMPTY_PERSON_LIST = Collections.emptyList();
     private static final Set<String> EMPTY_STRING_SET = Collections.emptySet();
 
-    @Test
+    @org.junit.jupiter.api.Test
```

Comment:
What does this change do?

Reply:
@gautamrajulu  how do you declare that a method is a test method?

Reply:
@ren jie  why do you need a new junit framework for this enhancement?

Reply:
@arhjaye  why do you need another junit test framework for your enhancement?